src/main/java/com/rachelbock/resources/CompletedResource.java

```diff
@@ -103,6 +103,13 @@ public boolean removeCompletedClimb(@PathParam("username") String username, @Pat
                 climb.setTapeColor(resultSet.getString("tape_color"));
                 climb.setType(resultSet.getString("climb_type"));
                 climb.setCompleted(true);
+                if (resultSet.getBoolean("removed")) {
```

Comment:
this can actually be simplified to `climb.setRemoved( resultSet.getBoolean("removed") )` if you want, but not required